src/audio/audio.cpp

```diff
@@ -40,15 +37,14 @@ Audio::State NoAudio::Update()
 	return Audio::State::NONE;
 }
 
-std::unique_ptr<Response> NoAudio::Emit(const std::string &path, bool)
+std::pair<std::string, std::string> NoAudio::Emit(const std::string &path)
 {
-	std::unique_ptr<Response> ret;
-
-	if (path == "/control/state") {
-		ret = Response::Res("Entry", "/control/state", "Ejected");
+	if (path == "/player/state/current" || path == "/player/state/available") {
+		// Only possible value is the set value
+		return std::make_pair("Entry", "ejected");
```

Comment:
Seems strange that this isn't `AUDIO::STATE_STRINGS[Audio::State::NONE]`, in terms of DRY